WebDevStudios/wds-acf-blocks
# Feature/#28 Block Cleanup

Closes #28 

### DESCRIPTION ###
- Cleanup ACF block and remove old blocks.
- The plugin is now similar to the `WDS Blocks` plugin where we only have the `Accordion` and `Carousel` blocks as base blocks.
- Removed `slick-slider` and replaced with `glider.js`, `Carousel` block now uses this, the only styling added is the base style from `Glider.js`
- `Accordion` is now accessible and transition is smoother.
- Left all other functions as they might be useful.
- ACF blocks are set to `'mode' => 'auto',`. Blocks will show as preview by default and clicking on the block will trigger the edit mode. No more edit fields in the sidebar which is just confusing and clunky.
- Updated README with new screenshots and dev instructions.
- ACF json will save on the theme by default, an `acf-json` folder needs to be created/transferred to the theme along with the 3 json files that contain the Accordion, Carousel, and the Button - Clone ACF groups.
- A base style for the `Accordion` block is also added in but will need to be added to the theme instead, it is already using `Tailwind` and can be used as a starter to style the accordion as needed.

### SCREENSHOTS ###

https://user-images.githubusercontent.com/1948204/122466943-7fe6dd80-cfec-11eb-8a35-b6a635899c47.mp4



### OTHER ###

### STEPS TO VERIFY ###
1. Transfer the `acf-json` folder and the `_accordion.scss` file to the theme.
2. Add either the `accordion` or the `carousel` block or both to test.
3. Check if both blocks are working.
